README.md

```diff
@@ -10,6 +10,10 @@
 
 For more information: https://hackmd.io/@Vid201/aa-bundler-rust
 
+## Prerequisites
+
+1. [install solc](https://docs.soliditylang.org/en/v0.8.17/installing-solidity.html) >=0.8.12
```

Comment:
@Vid201 I am curious whether `protoc` is needed. I tried and it seems unnecessary.

Reply:
It's not necessary, no need to add that.